### PR TITLE
Add posting and fetching score features

### PR DESCRIPTION
**Pull request by ntazelo**
### Functionality added

**posting score using an API**
**fetching score using an API**

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @ntazelo 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/addRes.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
-export default function styleResField() {
-  const resField = document.querySelectorAll('.cont-res-dis li');
-
-  resField.forEach((el, i) => {
+export default function populate(data) {
+  const list = document.querySelector('.cont-res-dis');
+  list.innerHTML = null;
+  data.forEach((el, i) => {
+    const userScore = document.createElement('li');
+    userScore.className = 'user-scr';
```
Comment (mikenath223) [inline src/addRes.js]: - [ ] As stated in the Javascript best practices guide please change all regular functions to es6 arrow function syntax.

File: src/addScore.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import getScore from './getScore';
+
+const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const id = 'f4qNRrUKtWgEVFl8Y7jY';
+const urlPost = `${URL}${id}/scores/`;
+
+export default function addPlayer() {
+  const formSubm = document.querySelector('.form');
+  const nameField = document.querySelector('.usr-name');
+  const scoreField = document.querySelector('.usr-score');
```
Comment (mikenath223) [inline src/addScore.js]: - [ ] Kindly change this to es6 arrow function syntax.

File: src/getScore.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const id = 'f4qNRrUKtWgEVFl8Y7jY';
+const urlPost = `${URL}${id}/scores/`;
+
+export default async function getScore() {
+  const data = await axios.get(urlPost);
+  return data;
```
Comment (mikenath223) [inline src/getScore.js]: - [ ] Kindly change this to es6 arrow function syntax.

File: src/addRes.js (hunk 1 above)
Comment (rloterh) [reply src/addRes.js]: This issue is well resolved! ✔️

File: src/addScore.js (hunk 2 above)
Comment (rloterh) [reply src/addScore.js]: This issue is well resolved! ✔️

File: src/getScore.js (hunk 3 above)
Comment (rloterh) [reply src/getScore.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Eloi, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**